script/test/default

```diff
@@ -3,17 +3,18 @@
 
 set -ex
 
-TAG="docker-compose:$(git rev-parse --short HEAD)"
+TAG="docker-compose:alpine-$(git rev-parse --short HEAD)"
 
-# By default use the Dockerfile, but can be overridden to use an alternative file
+# By default use the Dockerfile.alpine, but can be overridden to use an alternative file
 # e.g DOCKERFILE=Dockerfile.armhf script/test/default
-DOCKERFILE="${DOCKERFILE:-Dockerfile}"
+DOCKERFILE="${DOCKERFILE:-Dockerfile.alpine}"
+DOCKER_BUILD_TARGET="${DOCKER_BUILD_TARGET:-build}"
 
 rm -rf coverage-html
 # Create the host directory so it's owned by $USER
 mkdir -p coverage-html
 
-docker build -f ${DOCKERFILE} -t "$TAG" .
+docker build -f ${DOCKERFILE} -t "${TAG}" --target "${DOCKER_BUILD_TARGET}" .
```

Comment:
It would be cleaner to use

docker build -f $DOCKERFILE -t "$TAG" --target "$DOCKER_BUILD_TARGET"

here. Even though I cannot explicitly find that space is forbidden in docker tags (https://docs.docker.com/engine/reference/commandline/tag/), is it really allowed? Otherwise, using $TAG instead of "$TAG" would be preferred.

Reply:
Here I just use the double quotes because the line before was using it.